data/publiccloud/restart_instance.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash -e
+
+check_ssh()
+{
+    nc -w1 -z $host 22
```

Comment:
nc -w1 -z $1 22

as $host is outside this function.

Reply:
my bad, this is correct. Shell variables have a dynamic scope. If a variable is declared as local to a function, that scope remains until the function returns.

Reply:
I changed it  anyway.